### DISCUSSION TITLE
returning bokeh chart from bt.plot()

### DISCUSSION THREAD

**Discussion by gaussianswan** [kernc/backtesting.py#291] [Q&A]
I know that we have the html saved when we do bt.plot(). Is it possible to get access to the underlying figure object when we do that command bt.plot()

**Comment by kernc** [comment]
`backtesting._plotting.plot()` returns the figure object:
https://github.com/kernc/backtesting.py/blob/73e1534428fc2db9e681030862133b5067508520/backtesting/_plotting.py#L712
But the interface/layout/series/plots is deemed unstable, so use at own risk.

What would you do with it?

**Reply by gaussianswan** [reply]
I wanted to use this with a stream lit application. Would be plotting the chart you create on the app but it needs the figure object to plot it
